README.md

```diff
@@ -237,18 +187,41 @@ T | make-series AvgHatInventory=avg(HatInventory) default=double(null) on Timest
   | extend series_decompose_forecast(AvgHatInventory, 30) | project-away *residual, *baseline, *seasonal
 ```
 
-### Trace Visualization Support
+### Query Builder
 
-The trace format option can be used to display appropriately formatted data using the built in trace visualization. To use this visualization, data must be presented following the schema that is defined [here](https://grafana.com/docs/grafana/latest/explore/trace-integration/#data-frame-structure). The schema contains the `logs`, `serviceTags`, and `tags` fields which are expected to be JSON objects. These fields will be converted to the expected data structure provided the schema in ADX matches the below:
+| Field | Description |
+| -- | -- |
+| Table | Select a table. |
+| Columns | Select a subset of columns for faster results. Time series requires both time and number values; other columns are rendered as dimensions. For more information about dimensions, refer to [Time series dimensions](https://grafana.com/docs/grafana/latest/fundamentals/timeseries-dimensions/). |
+| Filters | (Optional) Add filters for the selected columns. Values for filters will be restricted to the column's data type. |
+| Aggregate | (Optional) Add aggregations for the selected columns. Select an aggregation type from the dropdown and select a column to aggregate on.  |
+| Group by | (Optional) Add group bys for the selected columns. For time group bys select a time range bucket. |
+| Timeshift | (Optional) **DO NOT MERGE!** why does this exist? |
```

Comment:
is the Timeshift entry intentional?

Reply:
@aangelisc @asimpson Do y'all know?

Reply:
This was added in #142 and I understand what it does based on testing but tbh looking at [this](https://github.com/grafana/grafana/issues/24784#issuecomment-704160213) comment we may be able to remove this field entirely in future (once we migrate queries). 

In order to update the `README` appropriately I'd say something along the lines of `Shift the time ranges generated from Grafana macros by a predetermined duration` or similar.